packages/connect-explorer/src/pages/methods/other/init.mdx

```diff
@@ -33,14 +42,27 @@ export const paramDescriptions = {
     popup: 'default `true`. Projects running on trusted domains (trezor.io) are not required to use popup. For other domains this option is ignored.',
     transports:
         'Array of transports that should be use. If not provided, TrezorConnect will chose a reasonable default based on your environment.',
+    coreMode:
+        'default `iframe`. If set to `popup`, TrezorConnect will use popup for interaction with user. The advantage is that popup enables WebUSB to work cross-origin on the web. The disadvantage is that popup does not retain a stateful connection to the device, which can be a problem for some use-cases. If set to `auto` TrezorConnect will use iframe only if bridge is available.',
 };
 
 ## Init
 
-Initiate TrezorConnect
+Initiate TrezorConnect. It is important to set the manifest object with your email and appUrl, otherwise TrezorConnect methods won't be available.
+
+It is also possible to customize more advanced options for TrezorConnect.
```

Comment:
I would mention why we need this. Could @Hannsek step in please?

Reply:
something like this:

```suggestion
To initiate TrezorConnect, please provide the manifest object with your email and appURL. This is crucial for us to identify your use of TrezorConnect and to contact you if necessary.

If the manifest object is not set, TrezorConnect methods will not be available.

It is important to set the manifest object with your email and appUrl, otherwise TrezorConnect methods won't be available.

It is also possible to customize more advanced options for TrezorConnect.
```